### ISSUE TITLE
Using ssl_port without ssl => true makes module fail silently

### ISSUE THREAD

**Issue by mzhaase** [voxpupuli/puppet-nginx#1015]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet:              4.8.1
- Ruby:                  ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
- Distribution:       Ubuntu 16.04
- Module version: 0.6.0

## How to reproduce (e.g Puppet code you use)
```
class { 'nginx': }
nginx::resource::server {'<name>':
      www_root    => '/var/www/test',
      listen_port => 443,
      ssl_port    => 443,
}
```

## What are you seeing
This creates <name>.conf in /etc/nginx/sites-enabled which only contains:

```
location / {
    root      /var/www/test;
    index     index.html index.htm index.php;
  }
```

## What behaviour did you expect instead
Any kind of error message.

**Comment by wyardley** [comment]
I'm not sure if this is the right fix, but I'm proposing one in #1022

**Comment by wyardley** [comment]
#1022 has been merged; hopefully this is sufficient.
